test/acceptance/docker-compose.yaml

```diff
@@ -9,4 +9,9 @@ services:
       CHROME_ARGS: "--headless --ignore-certificate-errors"
       BROWSERSTACK_USERNAME: ${BROWSERSTACK_USERNAME}
       BROWSERSTACK_ACCESS_KEY: ${BROWSERSTACK_ACCESS_KEY}
-      BROWSERSTACK_PARALLEL_RUNS: ${BROWSERSTACK_PARALLEL_RUNS}
\ No newline at end of file
+      BROWSERSTACK_PARALLEL_RUNS: ${BROWSERSTACK_PARALLEL_RUNS}
+      JWT_SECRET: ${JWT_SECRET}
+      JWT_ISSUER: ${JWT_ISSUER}
+      NODE_TLS_REJECT_UNAUTHORIZED: "0"
+      NOTIFY_TEMPLATE_ID_FARMER_APPLY_LOGIN: ${NOTIFY_TEMPLATE_ID_FARMER_APPLY_LOGIN}
+      SERVICE_URI: ${SERVICE_URI}
```

Comment:
dont think we need `SERVICE_URI ` as we already have `TEST_ENVIRONMENT_ROOT_URL`

Reply:
Correct, and TEST_ENVIRONMENT_ROOT_URL is [already being pushed to jenkins ](https://github.com/DEFRA/ffc-jenkins-pipeline-library/blob/f1f75434faa46ba747731e7d10a628fecb2a78dd/src/uk/gov/defra/ffc/Tests.groovy#L207)